html5ever/examples/arena.rs

```diff
@@ -147,19 +152,26 @@ impl<'arena> Sink<'arena> {
     }
 
     fn append_common<P, A>(&self, child: NodeOrText<Ref<'arena>>, previous: P, append: A)
-        where P: FnOnce() -> Option<Ref<'arena>>,
-              A: FnOnce(Ref<'arena>),
+    where
+        P: FnOnce() -> Option<Ref<'arena>>,
+        A: FnOnce(Ref<'arena>),
     {
         let new_node = match child {
             NodeOrText::AppendText(text) => {
                 // Append to an existing Text node if we have one.
-                if let Some(&Node { data: NodeData::Text { ref contents }, .. }) = previous() {
+                if let Some(&Node {
+                    data: NodeData::Text { ref contents },
+                    ..
+                }) = previous()
+                {
```

Comment:
I'm not happy about the placement of some of the newlines, like the one here. Maybe I could manually adjust these cases.